data/projects.js

```diff
@@ -421,5 +421,23 @@ coala should support generating metrics for your code.",
 		"tags" : ["Jetbrains IDEs", "Plugins"],
 		"markdown": "jetbrains-ide-plugin.md",
 		"collaborating_projects" : ["coala", "jetbrains"]
+	},
+	{
+		"name" : "Vulture dead code removal",
+		"desc" : "This project is about using vulture to automatically remove \
+				  unused python code.",
+		"requirements" : [
+				"The participant should have one patch to \
+				[vulture](https://bitbucket.org/jendrikseipp/vulture) accepted.",
+				"Being familiar with the current Bear API and what it provides.",
+				"Knowledge of ASTs and basic language grammar is a bonus."
+		],
+		"difficulty" : "medium",
+		"issues" : [],
+		"mentors" : ["jendrikseipp"],
```

Comment:
can anybody help me with coala-bear medium level issues as .I have been working but nobody answers my comments.
I first red all the code of vulture repository.
I learned about the power of ast and it's uses.
I used and have some knowledge about pyflake and coverage
I just finished reading the basic code of coala-bears